client/library/library/audits/partydao-2.html

```diff
@@ -0,0 +1,66 @@
+<page
+  clientName="PartyDAO"
+  reportDate="Feb 7, 2022"
+  auditTitle="PartyDAO A-2"
+  auditVersion="1.0.0"
+  repoUrl="https://github.com/PartyDAO/party-protocol"
+  passwordEncrypt="env:PAGE_PASS_PARTYDAO_2"
+  layout="/library/audits/_layout.html"
+  customRepoInfo
+>
+
+<content-for name="schedule">
+  The security audit was performed by the Macro security team from January 16, 2023 to January 27, 2023.
+</content-for>
+
+<content-for name="spec">
+  <ul>
+    <li>Discussions on Telegram with the {{page.clientName}} team.</li>
+    <li>Available documentation in the pull request.</li>
+  </ul>
+</content-for>
+
+<content-for name="repo-info">
+    <ul>
+      <li>
+        <b>Pull Request:</b>
+        <a href="https://github.com/PartyDAO/party-protocol/pull/211" target="_blank">
+          Version 1.1
+        </a>
+      </li>
+      <li class="break-words break-all">
+        <b>Commit Hash:</b>
+        <code>991cf2f4bb9ff785a39a2fb4dd19900beaa2cfbc</code>
+      </li>
+      <li>
+        <b>Details: </b>
+        <ul>
```

Comment:
I'd add a note with something like:
Details:
These are new features and modifications implemented in the V1.1 version. 
Maybe detail which ones are features and mods?